analysis.md

```diff
@@ -0,0 +1,77 @@
+# Malware Analysis
+
+## Extracting the Source Code
+The file hash of the malicious document is `340795d1f2c2bdab1f2382188a7b5c838e0a79d3f059d2db9eb274b0205f6981`. I used [oletools](https://github.com/decalage2/oletools) to extract the VBA source code from the macro.
+
+## Layer 1
+The author of the script uses a lot of newline spacing to spread apart its contents. There is a function `ParsingA()` that appears to download content from the Internet, but it never seems to be used. You'll also notice that the author uses gratuitous string concatenation to confuse readers. For instance...
```

Comment:
The author uses newlines, but why? About how many? What's the visual effect?

There's an unused function, but why do you think it's there?

String concatenation confuses readers, true, but it exists because it's easy to do and and breaks simple string signatures. Like some yara rules might look for "\root\cimv2"